styles.css

```diff
@@ -20,6 +20,8 @@ a {
 
```

Comment:
- [ ] In Mobile screen Size please consider minimizing the `padding-top` from the `header` selector in your `CSS` and stick to the Figma design system as much as you can.

| Your Design      | Figma Design |
| ----------- | ----------- |
| ![image](https://user-images.githubusercontent.com/68030297/230239270-f8376969-e678-424d-a82d-b69b0e36b4f2.png)      | ![image](https://user-images.githubusercontent.com/68030297/230239607-6d8dac92-a24c-4a79-9fd4-e616fdd7057b.png)       |


### Consider the highlighted section in red as just a decorative image that has been added by the designers to indicate that this design is for mobile 👍🏼